docs/index.html

```diff
@@ -68,7 +68,7 @@ <h2 class="title">A 1x Engineer...</h2>
           <li>Thanks others for their time, effort, and energy.</li>
           <li>Can have colorful desktop backgrounds.</li>
           <li>Supports code in production, even if they did not write it.</li>
-          <li>Can feel like an imposter at times, and understands others may, too.</li>
+          <li>Can feel like an impostor at times, and understands others may, too.</li>
```

Comment:
```suggestion
          <li>Can feel like an imposter at times, and understands others may, too.</li>
```

Reply:
I checked dictionaries, and it only shows the spelling with `o`